products/sle/main.pm

```diff
@@ -927,7 +927,8 @@ sub load_consoletests {
         loadtest "console/xfce_gnome_deps";
     }
     if (!is_staging() && sle_version_at_least('12-SP2')) {
-        loadtest "console/zypper_lifecycle";
+        # This test uses serial console too much to be reliable on Hyper-V
+        loadtest "console/zypper_lifecycle" unless check_var('VIRSH_VMM_FAMILY', 'hyperv');
```

Comment:
Could you please add reference to this [poo#30613](https://progress.opensuse.org/issues/30613) in the comment, so we could re-enable it in case we get a fix?